README.md

```diff
@@ -1,6 +1,16 @@
-# knausj_talon
+# community
 
-Community-maintained [Talon](https://talonvoice.com/) command grammar.
+Voice command set for [Talon](https://talonvoice.com/), community-supported.
+
+> **Note**
+>
+> Originally called `knausj_talon`, after [its original creator :superhero:](https://github.com/knausj85).
+>
+> You might see the reference to the original name in places. If you had the old name cloned, you don't have to do anything; GitHub will redirect the old name.
+>
+> However, if you'd like, you can update the remote URL in your existing local repo:
+>
+> `cd ~/.talon/knausj_talon && git remote set-url origin https://github.com/talonhub/community`
```

Comment:
I don't think we want so much about the original at the top here. I'd put the update steps in a separate section / file and just link to it here so it doesn't distract / confuse new users

Reply:
Agree

Reply:
Done